run-tests.php

```diff
@@ -1706,7 +1706,7 @@ function run_worker(): void
             ]);
         }
 
-        return;
+        return true;
```

Comment:
You can add a return type to anonys. functions as well. The one on line 1702 is an example :)

Reply:
Did that in the, hopefully, final pass.